src/compiler/types.ts

```diff
@@ -1956,6 +1956,8 @@ namespace ts {
         target?: TypeParameter;  // Instantiation target
         /* @internal */
         mapper?: TypeMapper;     // Instantiation mapper
+        /* @internal */
+        resolvedApparentType: Type;

```

Comment:
Oh, I missed that it was doing caching. I expected some other code to read it but it looks like it's only to avoid re-computing the apparent type.